src/govuk/objects/_width-container.scss

```diff
@@ -2,9 +2,30 @@
 @import "../tools/all";
 @import "../helpers/all";
 
-@mixin govuk-width-container {
-  // Limit the width of the container to the page width
-  max-width: $govuk-page-width;
+////
+/// @group objects
+////
+
+/// Width container mixin
+///
+/// Used to create page width and custom width container classes.
+///
+/// @param {Number} $width [$govuk-page-width] - Width in units (defaults to px)
+///
+/// @example scss - Creating a 1200px wide container class
+///  .govuk-width-container--wide {
+///    @include govuk-width-container(1200px);
+///  }
+///
+/// @access public
+
+@mixin govuk-width-container($width: $govuk-page-width) {
+  @if (unitless($width)) {
```

Comment:
Is this necessary to handle? Should we expect our users to explicitly set their units like they would elsewhere in their CSS?

Reply:
I think we should do the minimum the solve the problem but I don't think this is a problem to keep around if you think it's useful.

Reply:
Thanks for the IRL 🙌 Decided to do the very minimum here so have removed the unit conversion.